config/initializers/pg_search.rb

```diff
@@ -0,0 +1,3 @@
+PgSearch.multisearch_options = {
+  :using => {tsearch: { prefix: true, dictionary: "english"}},
```

Comment:
I'll add this as an issue for the future. I think it may be easier with rails 4, have to enable postgres extensions etc.